.functional-tests.env

```diff
@@ -30,3 +30,4 @@ EQ_SUBMISSION_CONFIRMATION_BACKEND=log
 VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS=35
 SDS_API_BASE_URL=http://localhost:5003
 OIDC_TOKEN_BACKEND=local
+EQ_FUNCTIONAL_TEST_MAX_INSTANCES=1
```

Comment:
Is this needed here?

Reply:
OK, added just in case but will remove now.